To-Do-app/src/Components/PageScrollingInfinite.tsx

```diff
@@ -0,0 +1,86 @@
+import axios from "axios"
+import React, { useEffect, useState } from "react"
+import InfiniteScroll from "react-infinite-scroll-component"
+
+interface IUser {
+    avatar: string
+    first_name: string
+    last_name: string
+    email: string
+}
+
+const PageScrollingInfinite = () => {
+
+    const Per_Page = 2
```

Comment:
variable names should not be starting with a capital letter.

Reply:
Ok sir will change that